airflow/providers/google/CHANGELOG.rst

```diff
@@ -148,6 +148,66 @@ Breaking changes
 
 * ``BigQueryHook.run_grant_dataset_view_access`` Remove ``source_project``. Please use ``project_id``
 
+Details of breaking changes:
+
+* ``'DataprocHook': Remove deprecated function 'submit' (#23389)``
+* ``'S3ToGCSOperator': Remove 'dest_gcs_conn_id' (#23348)``
+* ``Dataproc: Remove default value of 'region' (#23350)``
+* ``Google provider: Remove 'bigquery_conn_id', 'google_cloud_storage_conn_id' (#23326)``
+* ``'GCSFileTransformOperator': New templated fields 'source_object', 'destination_object' (#23328)``
+* ``'DatastoreHook': Remove 'datastore_conn_id' (#23323)``
+* ``'CloudBuildCreateBuildOperator': Remove deprecated 'body' parameter (#23263)``
+* ``Remove deprecated parameters from PubSub operators: (#23261)``
+* ``Remove deprecated parameters from BigQueryHook: (#23269)``
+* ``* 'CloudDatastoreImportEntitiesOperator' : Remove 'xcom_push'. Please use 'BaseOperator.do_xcom_push' (#23252)``
+* ``'BigtableCreateInstanceOperator' & 'BigtableHook.create_instance' Remove 'replica_cluster_id', 'replica_cluster_zone'. (#23251)``
+* ``Dataproc : remove 'location' in favor of 'region' (#23250)``
+* ``'PubSubPullSensor': Remove 'project' and 'return_immediately' (#23231)``
+* ``Remove deprecated 'params' from google operators (#23230)``
+
+Features
+~~~~~~~~
+
+* ``[FEATURE] google provider - split GkeStartPodOperator execute (#23518)``
+* ``Add exportContext.offload flag to CLOUD_SQL_EXPORT_VALIDATION. (#23614)``
+* ``Create links for Biqtable operators (#23164)``
```

Comment:
```suggestion
* ``Create links for Bigtable operators (#23164)``
```

Thoguh I dont know If we can change the messages ? Will it be overritten next time script runs?

Reply:
In the changelog no - we only check if the PR # is there already so we can freely modify them

The full override only happens in COMMITS.rst